docs/kintoapps.md

```diff
@@ -93,11 +93,42 @@ Be sure to include in the authorization header. Example:
 
 To make a call to gRPC, talk to grpc.prod.kintohub.com with your gRPC service proto file, but include in the grpc-meta-data the authorization header as well.
 
+## Viewing KintoApp Logs
+
+You can access the last 100 logs of an environment your KintoApp is deployed in, by going to *Environments* via the short cut menu on your KintoApp card, or from the KintoApp Manage page. Once there, open the edit page, and select *View Logs* on the top right hand side.
+
+The logs will give you information on the following things: 
+- Response code ie: 200, 500 etc.
+- Duration of the call
+- Which endpoint was called
+- Which KintoBlock was called
+- Which version of the KintoBlock was called
+- The time and date of the call
+
+If you wish to view more details of the call, these can be found by expanding that particular calls row in the table. Here you can view the JSON of any logs you have made. 
+
+### Required Format for Console Logs
+
+In order to view Console Logs in our system, specific syntax must be used. Every call to the block will container a header called `kinto-request-id`. This header will contain a value that needs to be sent back to us in order to associate the console log with the correct call. It is important to send your information formatted in JSON, with the correct quotation marks like this : `"key"`. An example for a *Node.js / Express* app would look like this:
```

Comment:
typo `will container` i think u meant `will contain`

Reply:
> This header will contain a value that needs to be sent back to us

I think that is confusing, its not gonna be send back in the response or anything, i would say it just needs to be as part of the log instead of sent back